ci/scripts/driver.sh

```diff
@@ -99,6 +109,8 @@ for pr in ${pr_list}; do
     for yaml_config in "${HOMEgfs}/ci/cases/"*.yaml; do
       pslot=$(basename "${yaml_config}" .yaml) || true
       export pslot
+      sed -i "/^base:/a\  ACCOUNT: \${SLURM_ACCOUNT}" "${pr_dir}/global-workflow/parm/config/gfs/yaml/defaults.yaml"
+      sed -i "/^base:/a\  ACCOUNT: \${SLURM_ACCOUNT}" "${pr_dir}/global-workflow/parm/config/gefs/yaml/defaults.yaml"
```

Comment:
Works for now, but I'm not a fan of needing to insert text into a file like this.

Reply:
I agree, we should write a feature change for "overriding" config values at the command prompt. **default.yaml** has required values in it.  We should have a switch that takes just user over rides.

Reply:
Added #1615 to use YAML inc feature.